examples/helpers/ldtk.rs

```diff
@@ -46,14 +51,34 @@ pub struct LdtkMapBundle {
 
 pub struct LdtkLoader;
 
+#[derive(Debug, Error)]
+pub enum LdtkAssetLoaderError {
+    /// An [IO](std::io) Error
+    #[error("Could not load LDTk file: {0}")]
+    Io(#[from] std::io::Error),
+}
+
 impl AssetLoader for LdtkLoader {
+    type Asset = LdtkMap;
+    type Settings = ();
+    type Error = LdtkAssetLoaderError;
+
     fn load<'a>(
         &'a self,
-        bytes: &'a [u8],
+        reader: &'a mut Reader,
+        _settings: &'a Self::Settings,
         load_context: &'a mut LoadContext,
-    ) -> BoxedFuture<'a, Result<(), anyhow::Error>> {
+    ) -> BoxedFuture<'a, Result<LdtkMap, LdtkAssetLoaderError>> {
```

Comment:
This can be `Self::Error` (same suggestion in the `Tiled` example)

Reply:
Oh, nice, that'll clean things up. Thanks!